.github/workflows/new-cvp-integration.yml

```
@@ -0,0 +1,74 @@
+---
+name: "CV integration testing"
+
+# on: pull_request
+on: workflow_dispatch
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+jobs:
+  # ----------------------------------------------- #
+  # CV INTEGRATION MOLECULE TEST FOR CV_WORKFLOW    #
+  # ----------------------------------------------- #
+  molecule_cv_workflow:
+    name: Run molecule test for new cvp integration
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        avd_scenario:
+          - 'cv_workflow'
+        ansible_version:
+          - 'ansible-core<2.17.0 --upgrade'
+    steps:
+      - name: 'Set environment variables'
+        run: |
+          echo "PY_COLORS=1" >> $GITHUB_ENV
+          echo "ANSIBLE_FORCE_COLOR=1" >> $GITHUB_ENV
+      - uses: actions/checkout@v3
```

There should be a newer version out
```suggestion
      - uses: actions/checkout@v4
```